### DISCUSSION TITLE
Charge timer

### DISCUSSION THREAD

**Discussion by openui54u** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#94] [Ideas]
Time your battery to be at x% at a certain time/weekplan. 
Perhaps even based on Agenda?
Easy for planned through the night charging. 
This way you can plan your battery charge status for weekend / weekdays, time.

**Comment by keiwanjamaly** [comment]
I can agree, that this is really helpful. I'm a student, and I'm bringing my Laptop to university without a charger, since the MacBook can easily run a full day without charging. But when I set it to top out it charges to max already in the evening, which keeps the battery at max for around 10+ hours, where it could be safely staying around 50% or so.

**Comment by kentzh1** [comment]
Regular charging is very helpful for regular work every day, or charging before planning to travel. I think it is more constructive
